crates/bw/src/main.rs

```diff
@@ -62,6 +68,39 @@ enum ItemCommands {
     Create {},
 }
 
+#[derive(Subcommand, Clone)]
+enum GeneratorCommands {
+    Password(PasswordGeneratorArgs),
+    Passphrase {},
+}
+
+#[derive(Args, Clone)]
+struct PasswordGeneratorArgs {
+    #[arg(short = 'l', long, action, help = "Include lowercase characters (a-z)")]
+    lowercase: bool,
+
+    #[arg(short = 'u', long, action, help = "Include uppercase characters (A-Z)")]
+    uppercase: bool,
+
+    #[arg(short = 'n', long, action, help = "Include numbers (0-9)")]
+    numbers: bool,
```

Comment:
Not sure about these parameters, by default they are all set to false, so running `bw generate password` we won't have any character types selected, which would probably have to return an error. 

You'll have to go out of your way to do `bw generate password -l -u -n` to get an alphanumeric password.

I think having a reasonable default for `bw generate password` is a good idea. Maybe we should invert the flags and make them `no-lowercase`, `no-uppercase` and `no-numbers`?

Reply:
My thinking is that if all are false the default applies. Which is how the CLI currently works.